tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -367,6 +369,72 @@ async def test_softmax_normalization(
     assert parse_data.get("intent") == intent_ranking[0]
 
 
+@pytest.mark.parametrize(
```

Comment:
don't we also need tests for the response selector / `TEDPolicy`?

Reply:
Added 👍